src/compiler/diagnosticMessages.json

```
@@ -6027,6 +6027,11 @@
         "category": "Error",
         "code": 7057
     },
+    "Try adding a new declaration (.d.ts) file containing `declare module '{0}';`": {

```

I added the new message based on the old one described in the issue ("Try `npm i --save-dev @types/foo` if it exists or add a new declaration (.d.ts) file containing `declare module 'foo/other'`;"), but without the "Try npm..." part.

---

Suggestion: I really like the “If the '{0}' package actually exposes this module” from the message you get when the types package exists. If we’ve detected other .d.ts files when issuing this message, we have reason to be at least a _little_ skeptical that the referenced module does in fact exist at runtime here. It might be nice to hedge our suggestion and make sure the user knows that this is a solution only if they’re really sure that this module exists.

```suggestion
    "If the '{0}' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module '{1}';`": {
```